packages/analytics/src/models/quests.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable object-shorthand */
```

Comment:
I think you can merge these three functions to one class, reuse the result from questList to get totalQuest and totalCompleted